packages/core/src/awsexplorer/activationShared.ts

```diff
@@ -43,9 +27,6 @@ export function registerToolView(viewNode: ToolView, context: vscode.ExtensionCo
             // Legacy CDK metric, remove this when we add something generic
             const recordExpandCdkOnce = once(() => telemetry.cdk_appExpanded.emit())
             recordExpandCdkOnce()
-        } else if (e.element.resource instanceof CodeWhispererNode) {
-            const onDidExpandCodeWhisperer = once(() => telemetry.ui_click.emit({ elementId: 'cw_parentNode' }))
```

Comment:
do we need a telemetry event for opening the CW status bar?

Reply:
Do you mean, as a opposed to `ui_click`? Seems worth having some sort of metric to capture clicking (or triggering) on the statusbar item, if we don't already have one. Can use `ui_click` if there isn't a more specific metric.

Reply:
added to when the status bar menu is opened.